elasticai/creator/examples/generate_vhdl_files/generate_all_vhd_files_for_lstm_cell.py

```diff
@@ -186,10 +184,3 @@ def define_lstm_cell(input_size: int, hidden_size: int) -> QLSTMCell:
     format_vhdl(file_path=file_path_sigmoid)
     format_vhdl(file_path=file_path_tanh)
     format_vhdl(file_path=file_path_testbench)
-
-    ### copy static files ###
-    for filename in os.listdir(ROOT_DIR + "/vhd_files/static_files/"):
-        shutil.copy(
-            ROOT_DIR + "/vhd_files/static_files/" + filename,
-            ROOT_DIR + "/" + args.path,
-        )
```

Comment:
Just copying all static files seems like a mighty bad idea. But since this is just an example i put it back into the code.
However as we already discussed, how the files are generated should be transparent to the user. Additionally i do not think we should distinguish between static files and templates. What we used to call a static file, is just a template without variables. The generation of each source file should be hidden behind a corresponding class, that way we can easily change how we generate them.